CONTRIBUTING.md

```diff
@@ -173,26 +181,27 @@ python tests/unit/unit_test_data_setup.py
 npm run test
 ```
 
-#### Step 4: Rebuild the package if necessary
+## Step 4: Rebuild the package if necessary
 
 If you have made changes to the JS code, then you need to rebuild the package:
 
 ```bash
-npm run build
+npm run build:all
```

Comment:
Not for this PR but just because I'm noticing it: unless we already have one, please make an issue to update our package scripts for dash-bio, using perhaps dcc as a model: there should only be three build scripts: `build:js`, `build:backends` (builds Py, R, and Jl using `dash-generate-components`, no `python -c`, no `get_version_info.py`), and `build` to combine them. Assuming our source mapping is set up correctly there is no reason for `build:dev` or `build:js-dev`.